src/commcare_cloud/ansible/deploy_prometheus.yml

```diff
@@ -0,0 +1,41 @@
+- name: Encrypt Data Drive
+  hosts: control1
+  become: true
+  roles:
+    - {role: ecryptfs, tags: 'ecryptfs'}
+
+- name: Prometheus
+  hosts: prometheus
+  become: true
+  tasks:
+    - include_role:
+        name: cloudalchemy.prometheus
```

Comment:
can't this be made a dependency of the dimagi role?

Reply:
Yeah , it can be done.